app/test/controllers/Todo/update.js

```diff
@@ -96,7 +77,26 @@ describe('Todos', () => {
                 .set('Authorization', 'Bearer ' + loggedUserToken)
                 .send({ name: updatedName });
 
-            expect(response.statusCode).to.equal(403);
+            expect(response.statusCode).to.equal(StatusCodes.FORBIDDEN);
+        });
+
+        it('Returns NOT_FOUND when todo does not exist', async () => {
+            const todo = await TodoFactory.build({
+                id: 666,
```

Comment:
not sure if we should use this number :D

Reply:
just generate any UUID

Reply:
want to fix it in next pr.  - I added UUID validator, so response will be different